src/Sylius/Bundle/ApiBundle/Resources/config/validation/CompleteOrder.xml

```diff
@@ -41,6 +41,11 @@
                 <value>sylius_checkout_complete</value>
             </option>
         </constraint>
+        <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\OrderShippingMethodAvailable">
```

Comment:
```suggestion
        <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\OrderShippingMethodAvailability">
```
to be consistent with other constraints

Reply:
I moved the logic to the Eligibility validator.